CHANGELOG.md

```diff
@@ -4,6 +4,24 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.45.0
+
+- build(cdn): Ensure ES5 bundles do not use non-ES5 code (#7550)
```

Comment:
we usually don't include build stuff but since this fixes a bug, let's leave it in